addons/block_code/ui/block_canvas/block_canvas.gd

```diff
@@ -7,9 +7,25 @@ const EXTEND_MARGIN: float = 800
 @onready var _window: Control = %Window
 @onready var _window_scroll: ScrollContainer = %WindowScroll
 
+var _block_scenes_by_class = {}
+
 signal reconnect_block(block: Block)
 
 
+func _ready():
+	_populate_block_scenes_by_class()
+
+
+func _populate_block_scenes_by_class():
+	for _class in ProjectSettings.get_global_class_list():
+		if not _class.base.ends_with("Block"):
```

Comment:
I don't like this being a substring match. Can we instead use `ClassDB.is_parent_class` https://docs.godotengine.org/en/stable/classes/class_classdb.html#class-classdb-method-is-parent-class to see if the class is a subclass of Block?

Reply:
Good point! Checking...

Reply:
I tried comparing like this:
```
		if not ClassDB.is_parent_class(_class.base, &"Block"):
			continue
```
And with "Block" (no &).. and it didn't work :(